azure-pipelines.yml

```diff
@@ -23,6 +23,11 @@ resources:
   - container: centos6_x64_build_image
     image: microsoft/dotnet-buildtools-prereqs:centos-6-376e1a3-20174311014331
 
+trigger: none
```

Comment:
Won't this disable ci triggers? (Is that the intention)?

Reply:
Yes, let's do transition in several steps - first PR validation, then CI